apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/DatastoreDbImpl.cs

```diff
@@ -277,6 +349,6 @@ private static void PropagateKeys<T>(IList<T> values, CommitResponse response, A
         private static void SetKey(Entity entity, Key key) => entity.Key = key;
 
         private static ReadOptions GetReadOptions(ReadConsistency? readConsistency) =>
-            readConsistency == null ? null : new ReadOptions { ReadConsistency = readConsistency.Value };
+            readConsistency == null ? null : new ReadOptions { ReadConsistency = readConsistency.Value };

```

Comment:
What is the difference here?

Reply:
Might be a rogue carriage-return, either in the original or in the new version?